src/getClientToken.ts

```diff
@@ -1,22 +1,19 @@
-import { DesktopIAPOptions } from "./types";
+/// <reference lib="dom" />
```

Comment:
why's this needed?

Reply:
This is leftovers of something I tried, thank you for catching it, I'll remove it.